app/templates/components/tables/events/list.hbs

```diff
@@ -0,0 +1,16 @@
+{{#ember-table class="ui table" as |t|}}
+  {{t.head columns=columns}}
```

Comment:
We don't want to put all of this boilerplate code for every table we want to render. So move this out to a component. And then call that component here with the rows, columns.